sources/pebble_vm_provision_as_root.sh

```diff
@@ -13,9 +12,10 @@ echo "                     Upgrading                     "
 echo "---------------------------------------------------"
 sudo DEBIAN_FRONTEND=noninteractive apt-get --assume-yes --option Dpkg::Options::="--force-confdef" --option Dpkg::Options::="--force-confold" upgrade
 echo "---------------------------------------------------"
-echo "                     Installing                    "
+echo "           Installing Dev Dependencies             "
 echo "---------------------------------------------------"
-sudo apt-get install --assume-yes git-core python-pip python2.7-dev language-pack-en-base
+sudo apt-get install --assume-yes git-core python-pip python2.7-dev language-pack-en-base \
```

Comment:
If you're editing these anyways, can you make this one-per-line? it makes diffs easier to read.

If you don't get to it, I'll do it later.